backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -61,6 +66,18 @@ private async void OnMissionUpdate(object? sender, MqttReceivedArgs mqttArgs)
                     mission.Status,
                     mission.RobotId
                 );
+            if (mission.Status == "completed")
+                _logger.LogInformation("Mission with ISAR mission id {id} is completed by the robot {name}", mission.MissionId, mission.RobotId);
```

Comment:
Based on the previous commits, im confused if this is now ISAR mission ID or Echo mission ID ?